hoomd/hpmc/test-py/hpmc_gsd_state.py

```diff
@@ -371,5 +371,18 @@ def test_convex_polyhedron(self):
         with self.assertRaises(RuntimeError):
             self.mc = hpmc.integrate.convex_polyhedron(seed=2234, d=0.3, a=0.4, restore_state=True);
 
+    def test_sphere_union(self):
```

Comment:
What is this doing here? This test doesn't even run on this branch. GSD shape/state writing could bet tested for all valid shape args in `test_shape.py`.